addons/binding/org.openhab.binding.velux/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please include the build in the parent `pom.xml` in that case it will already be tested and analysed by travis. And it will become available in [jfrog builds list](https://openhab.jfrog.io/openhab/libs-pullrequest-local/org/openhab/binding/).

Please also check:

http://docs.openhab.org/developers/development/bindings.html
http://docs.openhab.org/developers/development/guidelines.html

Reply:
done.